martin/src/pg/function_source.rs

```diff
@@ -33,6 +34,18 @@ pub async fn query_available_function(pool: &PgPool) -> Result<SqlFuncInfoMapMap
             let output_record_names = jsonb_to_vec(&row.get("output_record_names"));
             let input_types = jsonb_to_vec(&row.get("input_types")).expect("Can't get input types");
             let input_names = jsonb_to_vec(&row.get("input_names")).expect("Can't get input names");
+            let tilejson = if let Some(text) = row.get("description") {
+                match serde_json::from_str::<Value>(text) {
```

Comment:
parsing now works because it now uses `from_str::<Value>` instead of `from_str::<TileJSON>`.  Value allows any valid json.

Reply:
Yeah. hehe. I realized that lol. Cool. ✅